### PR TITLE
Document change for Release30Apr2021

### PR DESCRIPTION
**Pull request by nibinqtl**

### CODE REVIEW

**Comment by nibinqtl** [comment]
@burtonquantil or @marcfishelquantil , can you make a final review of this PR?

File: docs/edge-logic/supported-directives.md [hunk 1]
```diff
@@ -464,9 +466,9 @@ This is a wrapper of the [proxy_set_header](http://nginx.org/en/docs/http/ngx_ht
 
 Because of the hierarchical cache structure, the built-in variables $scheme and $remote_addr cannot be used. If you need to pass the scheme or IP address used by the client to the origin servers, use the following variables:
 
-*   $request_scheme: scheme used by the client
-*   $client_real_ip:  client’s IP address
-*   $client_country_code:  client’s ISO3166 country code
+*   [$request_scheme](/cdn/docs/edge-logic/built-in-variables#request_scheme): scheme used by the client
+*   [$client_real_ip](/cdn/docs/edge-logic/built-in-variables#client_real_ip):  client’s IP address
+*   [$client_country_code](/cdn/docs/edge-logic/built-in-variables#client_country_code):  client’s ISO3166 country code
```
Comment (burtonquantil) [inline docs/edge-logic/supported-directives.md:471]: Replace "ISO3166" with "ISO 3166" .

File: docs/edge-logic/supported-directives.md [hunk 2]
```diff
@@ -484,6 +486,16 @@ One thing to notice is that if you want to use this directive to set the `Host`
...
 
+### [`proxy_cache_background_update`](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_cache_background_update)
+
+<span class="badge">standard</span>
+
+**Syntax:** `proxy_cache_background_update on | off;;` <br/>
+**Default:** `proxy_cache_background_update off;` <br/>
+**Context:** server, location
+
+Turning it on allows a background subrequest to be fired to update an expired cache item, while a stale cached response is returned to the client. It should help with the responsiveness in serving popular large files, which might take a while to fetch from the origin. It should be used in conjunction with the [`proxy_cache_use_stale'](#proxy_cache_use_stale) directive with the `updating` option.
```
Comment (burtonquantil) [inline docs/edge-logic/supported-directives.md:497]: The commas aren't necessary. 
Also, I'd change "in serving" to "when serving"

**Review by burtonquantil — CHANGES_REQUESTED** [review]
Just a couple of small suggestions.

File: docs/edge-logic/supported-directives.md (hunk 1 above)
Comment (nibinqtl) [reply docs/edge-logic/supported-directives.md]: fixed

File: docs/edge-logic/supported-directives.md (hunk 2 above)
Comment (nibinqtl) [reply docs/edge-logic/supported-directives.md]: Thanks, fixed.

**Review by burtonquantil — APPROVED** [review]
OK to merge

**Pull request merged**